packages/ui-extensions/src/surfaces/admin/api/discount-function-settings/metafields.ts

```diff
@@ -0,0 +1,76 @@
+const supportedDefinitionTypes = [
```

Comment:
Does this need to be a `const`? I feel it might be ideal to keep this as a type to avoid increasing the runtime size of the library.